.eslintrc.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
     'no-underscore-dangle': 'off', // conflicts with external libraries
     'padding-line-between-statements': 'off',
     'prefer-named-capture-group': 'off',
+    'require-unicode-regexp': 'off', // sometimes we only want ascii
```

Comment:
I don't fully agree with this: https://github.com/solid/community-server/issues/378#issuecomment-737146010

Reply:
I have no strong opinion here, so just make sure you and @RubenVerborgh come to an agreement so I can merge this today 😉

Reply:
@joachimvh Just eslint-ignore for now, we can revisit in https://github.com/solid/community-server/issues/378

Reply:
Actually, I might have been mistaken and you still might want the `u` flag in your expression.